### PR TITLE
Remote: Add is_container property

### PR DESCRIPTION
**Pull request by zmc**
Signed-off-by: Zack Cerza <zack@redhat.com>

### CODE REVIEW

File: teuthology/orchestra/remote.py [hunk 1]
```diff
@@ -679,6 +679,15 @@ def is_vm(self):
             self._is_vm = teuthology.lock.query.is_vm(self.name)
         return self._is_vm
 
+    @property
+    def is_container(self):
+        if not hasattr(self, '_is_container'):
+            self._is_container = not bool(self.run(
+                args="ls /.dockerenv || ls /run/.containerenv",
```
Comment (kshtsk) [inline teuthology/orchestra/remote.py:686]: ls?

Comment (zmc) [reply teuthology/orchestra/remote.py:686]: Just checking for the existence of either file.

Comment (kshtsk) [reply teuthology/orchestra/remote.py:686]: maybe `test` is better?

Comment (zmc) [reply teuthology/orchestra/remote.py]: Sure, updatedd

**Review by jdurgin — APPROVED** [review]

**Pull request merged**